src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,101 @@
+package task;
+
+import command.CommandException;
+import main.Main;
+import util.Storage;
+
+import java.util.ArrayList;
+import java.util.function.BiConsumer;
+
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+
+    /**
+     * The default constructor for the Task class
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Finds a task in the list by name and returns the task instance
+     *
+     * @param name The name of the task that you want to find
+     * @return The task instance found by the given name, or "null" if no task in the list has the given name.
+     */
+    public Task findTaskByName(String name) {
+        int count = this.tasks.size();
```

Comment:
```suggestion
        int count = tasks.size();
```
is "this" required here?